_posts/2023-01-11-cockpit-283.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cockpit 283
+author: garrett
+date: '2023-01-11'
+tags: cockpit, machines
+slug: cockpit-283
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 283 and cockpit-machines 281:
+
+
+## Services: Create timer to run every minute
```

Comment:
let's add special thanks to the upstream contributor for that. It's https://github.com/dphaas2004

Reply:
Great suggestion! It's always good to have shout-outs to contributors.